src/subapps/projects/components/ResourcesPane.tsx

```diff
@@ -9,10 +9,11 @@ const { Panel } = Collapse;
 
 import './ResourcesPane.less';
 
-const ResourcesPane: React.FC<{ linkCode(data: CodeResourceData): void }> = ({
-  children,
-  linkCode,
-}) => {
+const ResourcesPane: React.FC<{
```

Comment:
we don't use this component anymore but it has some useful logic that we will extract soon